packages/notification-center/src/hooks/use-default-theme.hook.ts

```diff
@@ -21,10 +23,10 @@ export function useDefaultTheme(props: IDefaultThemeProps): {
 } {
   const theme =
     props.colorScheme === 'light'
-      ? merge(defaultLightTheme, props?.theme?.light)
-      : merge(defaultDarkTheme, props?.theme?.dark);
+      ? merge(cloneDeep(defaultLightTheme), props?.theme?.light)
+      : merge(cloneDeep(defaultDarkTheme), props?.theme?.dark);
```

Comment:
there was actually a bug with the default theme objects... these objects were defined as immutable with `Object.freeze()`... but the freeze doesn't do deep freeze it only does it on the props, but not on deep leaves... 

the thing is that the lodash merge applies the changes to the first argument - default object, meaning it tries to change it... but it can't do that because its props are frozen, but it can change the leaves which are not frozen...

in this case, there was a frozen prop `defaultDarkTheme.loaderColor` and the changes were not applied to that prop...

---

I've removed the `Object.freeze()` and added the `cloneDeep` so we won't modify the default object...

Reply:
I see that it's supported by all major browsers: https://caniuse.com/?search=structuredClone
So maybe I'll use it ;) thanks! 🙌